platypus/commands.py

```diff
@@ -50,10 +49,10 @@ def compare(interest_fp, other_fp, output_dir='blast-results-compare',
         other database search results. If None is passed, it defaults to
         `[50]`.
     hits_to_first : bool, optional defaults to False
-        Outputs the labels and counts of the sequences being hit in the first
+        Outputs all the labels of the sequences being hit in the first
```

Comment:
Can you update this in `scripts/platypus`, specifically in descriptions in lines 56 and 59.